### PR TITLE
Feature/bs platform@3.1.4

### PR DESCRIPTION
**Pull request by idkjs**
**Pull Request Labels**

- [ ] feature
- [ ] blocking
- [ ] docs

added `graphql/ppx@0.2.3`(per @emmenko), `bs-platform@3.1.4`(per @emmenko) and example of deleting a selected id.

### CODE REVIEW

File: src/graphql-types/ReasonApolloQuery.re [hunk 1]
```diff
@@ -100,14 +100,11 @@ module Get = (Config: ReasonApolloTypes.Config) => {
             "pollInterval": pollInterval |> fromOption,
             "notifyOnNetworkStatusChange":
               notifyOnNetworkStatusChange
-              |> Js.Option.map((. b) => Js.Boolean.to_js_boolean(b))
+              |> Js.Option.map((. b) => b)
```
Comment (emmenko) [inline src/graphql-types/ReasonApolloQuery.re]: I think this row can now be removed. /cc @Gregoirevda ?

File: src/graphql-types/ReasonApolloQuery.re [hunk 2]
```diff
@@ -100,14 +100,11 @@ module Get = (Config: ReasonApolloTypes.Config) => {
...
               |> fromOption,
             "fetchPolicy": fetchPolicy |> fromOption,
             "errorPolicy": errorPolicy |> fromOption,
-            "ssr":
-              ssr
-              |> Js.Option.map((. b) => Js.Boolean.to_js_boolean(b))
-              |> fromOption,
+            "ssr": ssr |> Js.Option.map((. b) => b) |> fromOption,
```
Comment (emmenko) [inline src/graphql-types/ReasonApolloQuery.re]: Same here:  remove `Js.Option.map((. b) => b)`

File: examples/swapi/src/DeleteButton.re [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        deletePerson(id: $id) {
+          id
+        }
+      }
+|}
+];
+
+module DeletePersonMutation = ReasonApollo.CreateMutation(DeletePerson);
+
+let component = ReasonReact.statelessComponent("DeleteButton");
```
Comment (emmenko) [inline examples/swapi/src/DeleteButton.re]: Nitpick: `DeletePersonButton` might be more accurate.

File: src/graphql-types/ReasonApolloQuery.re (hunk 1 above)
Comment (idkjs) [reply src/graphql-types/ReasonApolloQuery.re]: wow. That is not in my version. I had remembered removing it and its not there. 

I guess I don't understand the request here. Clarification? 

Thank you!

Comment (emmenko) [reply src/graphql-types/ReasonApolloQuery.re]: Yes, correct (but I'd like @Gregoirevda to confirm it) 😇

Comment (idkjs) [reply src/graphql-types/ReasonApolloQuery.re]: removed

File: src/graphql-types/ReasonApolloQuery.re (hunk 2 above)
Comment (idkjs) [reply src/graphql-types/ReasonApolloQuery.re]: removed

File: src/graphql-types/ReasonApolloQuery.re (hunk 1 above)
Comment (idkjs) [reply src/graphql-types/ReasonApolloQuery.re]: Adding ssrMode to `Client.instance` like so, compiles without error:
```
let instance =
  ReasonApollo.createApolloClient(
    ~link=httpLink,
    ~cache=inMemoryCache,
    ~ssrMode=true,
    (),
  );
```

I dont know if this means it worked.

Comment (Gregoirevda) [reply src/graphql-types/ReasonApolloQuery.re]: Yes correct, since we don't need to cast it anymore, we don't need to map on it

**Comment by Gregoirevda** [comment]
This should be closed right @idkjs

**Pull request merged**